src/Facebook/InstantArticles/Elements/Element.php

```diff
@@ -55,10 +58,31 @@ protected function dangerouslyAppendUnescapedHTML($element, $content)
     /**
      * Auxiliary method to extract all Elements full qualified class name.
      *
-     * @return string The full qualified name of class
+     * @return string The full qualified name of class.
      */
     public static function getClassName()
     {
         return get_called_class();
     }
+
+    /**
+     * Method that checks if the element is valid, not empty. If !valid() it wont be rendered.
+     * @since v1.0.7
+     * @return boolean true for valid element, false otherwise.
+     */
+    public function isValid()
+    {
+        return true;
+    }
+
+    /**
+     * Method to create an empty fragment if @see isValid() is false in @see toDOMElement()
```

Comment:
You cannot use @see inside the description as you're doing here (and some other places).

Look: https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.pkg.html

> phpDocumentor tags are very similar to tags for the JavaDoc tool for Sun's Java Programming Language. Tags are only parsed if they are the first thing on a new line of a DocBlock. You may use the @ character freely throughout documents as long as it does not begin a new line.

Please simply mention the methods in the descriptions and add the @see tags below.